common/src/main/templates/io/netty/util/collection/KObjectHashMap.template

```diff
@@ -458,10 +487,108 @@ public class @K@ObjectHashMap<V> implements @K@ObjectMap<V>, Iterable<@K@ObjectM
         }
     }
 
+    @Override
+    public String toString() {
+        if (size == 0) {
+            return "{}";
+        }
+        StringBuilder sb = new StringBuilder(4 * size);
+        for (int i = 0; i < values.length; ++i) {
+            V value = values[i];
+            if (value != null) {
+                sb.append(sb.length() == 0 ? "{" : ", ");
```

Comment:
Can we unroll this `sb.length() == 0` check?  I think we just want to unconditionally append a `'{'` before the loop anyways right?